package.json

```diff
@@ -10,28 +10,37 @@
     "build:css": "postcss styles/tailwind.css -o styles/main.css",
     "build:watch": "postcss styles/tailwind.css -o styles/main.css --watch",
```

Comment:
Do we need to keep these around? My understanding is that this gets done automatically with Next.js' dev server

Reply:
If that's the case then I can just remove that in a future pr